user_guide/feature_contexts/hooking_into_the_test_process.rst

```diff
@@ -226,6 +234,34 @@ same as a usual scenario.
 ``@AfterScenario`` functions exactly the same way, being executed both after
 usual scenarios and outline examples.
 
+Feature Hooks
```

Comment:
``Suite > Feature > Scenario > Step`` is the meaningful order, and it is the existing order before your PR